.github/workflows/nightly.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       run: dotnet test -c Release -f net9.0 --no-build
     
     - name: dotnet pack
-      run: dotnet pack -c Release /p:Version=${{ steps.version.outputs.version }} -o drop/nuget
+      run: dotnet pack -c Release /p:Version=${{ steps.version.outputs.version }} /p:ApiCompatGenerateSuppressionFile=true -o drop/nuget

```

Comment:
should we also change `release.yml`?

Reply:
As far as I've confirmed.
This compatibility error seems occurred when running `dotnet pack` command with .NET 9 SDK.

So CI release build is not affected. (By removing .NET 7 supports)

---

When it need to resolve compatibility error occurred.
It need to update baseline version defined at `Docfx.App.csproj`.
https://github.com/dotnet/docfx/blob/main/src/Docfx.App/Docfx.App.csproj#L4